<?xml version="1.0"?>
<launch>
  <master auto="start"/>
   
  <node pkg="map_tools" type="map_node" respawn="false" name="map_node" output="screen" required="true">
    <param name="wall_file" value="$(find ras_maze_map)/maps/contest_maze.txt" />
    <rosparam file="$(find map_tools)/yaml/map.yaml" command="load" />
  </node>
  
</launch> 
